#3 Create Provider and Recipient Content

darnjo opened the pull request:
Recreating this PR as the branch was renamed to use a "semantic" format. 

The previous PR (#2) was closed, so the review comments are no longer on this branch, [but can be found here](https://github.com/RESOStandards/reso-certification-help/pull/2/files).

Jad212s committed:
Update providers.md

Endorsements section complete

Jad212s committed:
Update providers.md

Jad212s committed:
Update providers.md

Provider completed with bundling verbiage added.

Jad212s committed:
Update providers.md

Jad212s committed:
Update providers.md

Jad212s committed:
Update providers.md

Jad212s committed:
Update recipients.md

Recipients w/o screenshots

Jad212s committed:
Added Screenshot Placeholders

Screenshots locations are noted in between brackets.

Jad212s committed:
Added Recipient Screenshot Placeholders

Jad212s commented on docs/providers.md:
```diff
@@ -7,98 +7,147 @@ After logging in to [certification.reso.org](certification.reso.org), the endors
+Selecting **Notify** for either Data Dictionary or the Web API Core report will send a bundled review notification for both endorsements. The recipient will be able to review and approve both reports from a single link that directs them to the Certification Summary page.
 
-**Note:** the only time the bundled notification is available is when BOTH Data Dictionary and Web API are in Passed status. Otherwise, the email will only be for the endorsement selected.
+**Note:** The bundled notification is available ONLY when BOTH Data Dictionary and Web API are in **Passed** status. Otherwise, the email will only include the single endorsement that was selected for notification.
```
Rewritten

Jad212s commented on docs/recipients.md:
```diff
@@ -24,48 +29,64 @@ The Data Elements view shows your field and lookup counts and how they compare t
+**Seconds per 1K Records** - an estimate of how many seconds it might take to pull 1,000 records using pages of 100 records. It's based on the average response time found during data sampling and validation. Measured in seconds (s).
 
-**N/A** means that the given recipient has not completed Data Dictionary 1.7 payloads testing yet. Please contact RESO at certification@reso.org if you have any questions.
+**N/A** means that the given recipient has not completed Data Dictionary Availability 1.7 testing yet. Please contact RESO at certification@reso.org if you have any questions.
```
Lowercase "availability" and move behind "1.7".

Jad212s commented on docs/recipients.md:
```diff
@@ -24,48 +29,64 @@ The Data Elements view shows your field and lookup counts and how they compare t
 
 **Payload Size** - a measure of how much data was retrieved when sampling the selected resource. When All is selected, it's the average across all resources found, measured in megabytes (MB).
 
 **Response Time** - shows how long, on average, a request to the given server took to complete, measured in seconds(s).
```
Space before (s).

Jad212s replied:
Space before (s).

Jad212s commented on docs/recipients.md:
```diff
@@ -1,10 +1,12 @@
 # Notifications
-You will receive an email from certification@reso.org with links to review and approve your results. The body of the email will contain a review link for each endorsement submitted by your Provider.
+You will receive an email from certification@reso.org with link(s) to review and approve your results. The body of the email will contain a review link for the endorsements that were submitted by your Provider.
```
Should be "link" singular with the bundled configuration.

Jad212s commented on docs/recipients.md:
```diff
@@ -1,10 +1,12 @@
-You will receive an email from certification@reso.org with links to review and approve your results. The body of the email will contain a review link for each endorsement submitted by your Provider.
+You will receive an email from certification@reso.org with link(s) to review and approve your results. The body of the email will contain a review link for the endorsements that were submitted by your Provider.
 
 Select the review and approve results link below the endorsement you would like to review.
```
Rewrite as "Select the **REVIEW AND APPROVE RESULTS** link located below the endorsement(s) to be taken to the certification summary page."

Jad212s committed:
Added Provider Screenshot Placeholders

Jad212s committed:
Added Recipient Bundled Email Instructions

gsaxRESO committed:
Suggested Edits from G. Sax

I cleaned up some little things: typos, word choices, bolding, etc. Overall, I believe this reads pretty well now. I'm eager to see it with the screenshots.

gsaxRESO committed:
Suggested Edits from G. Sax

This is my second edit after doing providers.md. I believe I caught all the changes to repeated text from that guide and applied it here as well.

Jad212s committed:
Added Create Token Instructions

Jad212s committed:
Completed Suggested Changes to Recipients

gsaxRESO reviewed (approved):
I read through the Providers and Recipients guides and suggested several light changes that appear to have been accepted by Jason. I believe that we're in good shape here.

gsaxRESO commented:
GitHub protocol and terminology is still fairly new to me, so please forgive any indiscretions during my editing, review and approval. I'm learning as I go and researching GitHub more, since we intend to use it more as a team at RESO.
